dev-support/docker/pkg-resolver/install-common-pkgs.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#######
+# Install SpotBugs 4.2.2
+#######
+mkdir -p /opt/spotbugs &&
+  curl -L -s -S https://github.com/spotbugs/spotbugs/releases/download/4.2.2/spotbugs-4.2.2.tgz \
+    -o /opt/spotbugs.tgz &&
+  tar xzf /opt/spotbugs.tgz --strip-components 1 -C /opt/spotbugs &&
+  chmod +x /opt/spotbugs/bin/*
+
+#######
+# Install Boost 1.72 (1.71 ships with Focal)
+#######
+# hadolint ignore=DL3003
+mkdir -p /opt/boost-library &&
```

Comment:
Would it make sense to have a file for each of them?
I can see possible future differences depending on the version.

Reply:
I think it make sense @goiri . Different platforms have different set of libraries to be built and installed. Having different `.sh` files for each platform should simplify it. I'll make the change.

Reply:
Sounds good.